homeassistant/components/metoffice/sensor.py

```diff
@@ -115,7 +115,7 @@
         key="visibility_distance",
         name="Visibility distance",
         native_unit_of_measurement=UnitOfLength.KILOMETERS,
-        device_class=SensorDeviceClass.DISTANCE,
+        device_class=None,
```

Comment:
```suggestion
```
Why not remove it?

Reply:
Was just following the style of the rest of the code - a number of the other sensors have been done the same way. Is that wrong?

Reply:
I'd say remove all of the `device_class=None,` as they don't provide any benefit to state it explicitly imo

Reply:
Yes. Remove it.

Reply:
All done 👍